reconcile/utils/terrascript_aws_client.py

```diff
@@ -2438,7 +2446,7 @@ def populate_tf_resource_s3(self, spec: ExternalResourceSpec) -> aws_s3_bucket:
         output_name = output_prefix + "__endpoint"
         tf_resources.append(Output(output_name, value=endpoint))
 
-        sqs_identifier = common_values.get("sqs_identifier", None)
+        sqs_identifier = common_values.get("sqs_identifier")
```

Comment:
can inline with next if to indicate this only used in that branch:

```python
if sqs_identifier := common_values.get("sqs_identifier"):
```

same as 2 below changed lines for `filter_prefix` and `filter_suffix`

Reply:
i always forget about the walrus